src/validate.js

```diff
@@ -0,0 +1,30 @@
+const casesToConvert = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+function validate(text = '', toCase) {
+  const errors = [];
+
+  if (!text.length) {
```

Comment:
```suggestion
  if (!text) {
```
Empty string is falsy too